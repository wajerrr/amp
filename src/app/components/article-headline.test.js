import React from 'react';
import renderer from 'react-test-renderer';
import ArticleHeadline from './article-headline';

describe('ArticleHeadline', () => {
  it('should match snapshot without subheadline', () => {
    const tree = renderer
      .create(<ArticleHeadline>description</ArticleHeadline>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('should match snapshot with subheadline', () => {
    const tree = renderer
      .create(
        <ArticleHeadline subheadline="subheadline">headlione</ArticleHeadline>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
