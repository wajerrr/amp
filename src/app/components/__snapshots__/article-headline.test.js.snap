// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleHeadline should match snapshot with subheadline 1`] = `
<h1
  className="css-1ev9gmn"
>
  <span
    className="css-1ajtu19"
  >
    subheadline
  </span>
  <span
    className="css-1ar97ln"
  >
    headlione
  </span>
</h1>
`;

exports[`ArticleHeadline should match snapshot without subheadline 1`] = `
<h1
  className="css-1ev9gmn"
>
  
  <span
    className="css-1ar97ln"
  >
    description
  </span>
</h1>
`;
